tests/integration/test_design_import.py

```diff
@@ -240,3 +240,23 @@ def test_open_file(modeler: Modeler, tmp_path_factory: pytest.TempPathFactory):
         # .prt
         design2 = modeler.open_file("./tests/integration/files/import/disk1.prt")
         assert len(design2.bodies) == 1
+
+
+def test_design_insert(modeler: Modeler):
+    """Test creation of a design, sketching a circle, extruding it and then inserting a
```

Comment:
```suggestion
    """Test creation of a design, sketching a circle, extruding itn and then inserting a
```
If there is a summary table with method descriptions, this is never goign to fit!  LOL!